### PR TITLE
addHomepage

### PR DESCRIPTION
**Pull request by minatimar**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
-    </>
+    <StyledDiv>
+      <StyledImage>
+        <Image
+          src={logoHomepage}
+          alt="Essensretter Willkommen Logo"
+          width={300}
+          height={300}
+        />
+      </StyledImage>
```
Comment (marcterre) [inline pages/index.js:15]: you could try styling the image directly

File: pages/index.js [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
-            />
-          );
-        })}
-      </StyledList>
-
-      <StyledNavigation as="a" myUserID={myUserID} href={"/myDonations"}>
-        Ich möchte spenden!
-      </StyledNavigation>
-    </>
+    <StyledDiv>
```
Comment (marcterre) [inline pages/index.js]: use something like "Wrapper" for naming a styled div :)

**Review by marcterre — APPROVED** [review]
Nice small pr, left small recommendations. The rest lgtm! 👍

File: pages/index.js (hunk 2 above)
Comment (minatimar) [reply pages/index.js]: change it to wrapper :)

**Pull request merged**